### PR TITLE
fix syntax error in main.py

### PR DESCRIPTION
**Pull request by ripkurrle**

### CODE REVIEW

File: pokemonterminal/main.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ def daemon(time_stamp, pkmn_list):
 def slideshow(filtered, delay, changer_func):
     pid = os.fork()
     if pid > 0:
-        print(f"Starting slideshow with {len(filtered)}, pokemon " +
```
Comment (cclauss) [inline pokemonterminal/main.py]: These are not syntax errors on Python 3.6.  They are features!
* https://docs.python.org/3.6/whatsnew/3.6.html#new-features

**Comment by ripkurrle** [comment]
oh, ok thanks

**Comment by samosaara** [comment]
See also #95

**Comment by ripkurrle** [comment]
Oh thanks.. good read... That scenario was pretty much me upgrading from older version on Ubuntu 16.04 lts which still uses python 2, easy enough to fix for me. Great work on the project, still love it, my first fork and commit so interesting to see the process.

**Pull request closed without merging**